packages/react-mui-store-file-viewer/src/components/DeleteFileButton/BulkDeleteFilesButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { PangeaDeleteModal } from "@pangeacyber/react-mui-shared";
+import { FC } from "react";
+import { ObjectStore } from "../../types";
+import { Button, ButtonProps, IconButton } from "@mui/material";
+import DeleteIcon from "@mui/icons-material/Delete";
+
+import { useStoreFileViewerContext } from "../../hooks/context";
+
+interface Props {
+  selected: string[];
+  view?: "text" | "icon";
+  ButtonProps?: ButtonProps;
+  onClose?: () => void;
+}
+
+const BulkDeleteFilesButton: FC<Props> = ({
+  selected,
+  ButtonProps,
+  onClose = () => {},
+  view = "text",
+}) => {
+  const { apiRef, reload } = useStoreFileViewerContext();
+
+  return (
+    <PangeaDeleteModal
+      description={`Are you sure you wish to delete the ${selected.length} selected along with all versions which will be permanently removed from the secure file store.`}
```

Comment:
Do we want a question mark here?
```suggestion
      description={`Are you sure you wish to delete the ${selected.length} selected along with all versions which will be permanently removed from the secure file store?`}
```